### PR TITLE
Revert to Node 10.13.0

### PR DESCRIPTION
**Pull request by izakp**
Setting `server.keepAliveTimeout` in https://github.com/artsy/force/pull/3232 did not solve 504s on staging.artsy.net - fallback to reverting the Node upgrade that correlates with this behavior

screen shot 2018-12-20 at 4 56 24 pm

### CODE REVIEW

File: hokusai/staging.yml [hunk 1]
```diff
@@ -80,7 +78,7 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-access-log-emit-interval: '60'
     service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-name: 'artsy-elb-logs'
     service.beta.kubernetes.io/aws-load-balancer-access-log-s3-bucket-prefix: 'staging-force'
-    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '59'
+    service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '300'
```
Comment (joeyAghion) [inline hokusai/staging.yml]: I believe this is just reverting the change that didn't seem to improve things. Just confirming.

Comment (izakp) [reply hokusai/staging.yml]: Yeah @dleve123 suggested we go back to square-1 and set it as it was set before we started seeing errors on staging.  It's trivial to adjust this and the `KEEPALIVE_TIMEOUT` env var

**Comment by joeyAghion** [comment]
Interestingly [the .nvmrc file](https://github.com/artsy/force/blob/master/.nvmrc) still refers to 10.13.

**Comment by dleve123** [comment]
Whoops I didn't see the assignment email yet! Apologies for any workflow hurdles here.

**Pull request merged**